src/components/Dashboard/DashboardTable.vue

```diff
@@ -1,35 +1,52 @@
 <template>
-  <el-card :body-style="{ padding: 0 }">
-    <div class="card_header" slot="header">
-      <el-input
-        placeholder="Search"
-        prefix-icon="el-icon-search"
-        suffix-icon="el-icon-date"
-        v-model="filterInput"/>
+  <div class="card-content shadow">
+    <div class="card-content_header">
+      <el-row class="card-content_header-row">
+        <el-col :span="24">
+          <el-input
+            placeholder="Search"
+            prefix-icon="el-icon-search"
+            suffix-icon="el-icon-date"
+            v-model="filterInput"/>
+        </el-col>
+      </el-row>
     </div>
-    <div>
+    <div class="card-content_body">
       <el-row justify="center" class="table_header">
-        <el-col :span="8" class="table_header-title">Currency</el-col>
-        <el-col :span="8" class="table_header-title">Balance</el-col>
-        <el-col :span="8" class="table_header-title">Changes</el-col>
+        <el-col :span="10" class="table_header-title">Currency</el-col>
+        <el-col :span="5" class="table_header-title">Balance</el-col>
+        <el-col :span="9" class="table_header-title text-right">Changes</el-col>
       </el-row>
       <el-row class="table_body">
-        <div class="table_body-item" v-for="(value, index) in filteredPortfolio" :key="index">
-          <div class="table_body-content" @click="selectCrypto(value.asset)">
-            <el-col :span="8">{{ value | formatName }}</el-col>
-            <el-col :span="8">
-              <span class="balance">{{ value.price | formatBalance }}</span>
-            </el-col>
-            <el-col :span="8">
-              <span :class="[value.diff > 0 ? 'uptrend' : 'downtrend']">
-                {{ value | formatDiff }}
-              </span>
-            </el-col>
+        <div class="table_body-content">
```

Comment:
width=1200
![electron](https://user-images.githubusercontent.com/1365915/42798615-72e8b326-89cf-11e8-84bc-5c3f7f9860f6.png)

I guess this is because of `width: 104%` of `.table_body-content`.

Reply:
Do you mean that numbers are too near to right corner? Is it with width 1200px?

This is my picture, width 1200px. I didn't change anything
![image](https://user-images.githubusercontent.com/16295803/42802203-e133c868-89aa-11e8-91da-e4bc784b441f.png)